dessia_common/document_generator.py

```diff
@@ -148,6 +175,7 @@ def __init__(self, paragraphs: List[Paragraph] = None, section: Section = None,
         if paragraphs is None:
             paragraphs = []
         self.paragraphs = paragraphs
+        self.table = table
```

Comment:
After another look at the code, I find it weird that DocxWriter always have a table attribute. What if some document doesn't or has many tables ?

Reply:
I added a list

Reply:
by default table is None